docs/spec/staking/definitions and examples.md

```diff
@@ -108,7 +108,41 @@ worth more as share-to-atom-ex-rate is now worth 50/45 Atoms per share. Therefor
 
 `delegatorCoins = 10 (delegator shares) * 1 (delegator-share-to-global-share-ex-rate) * 50/45 (share-to-atom-ex-rate) = 100/9 Atoms`  
    
-
+### Inflation provisions
+
+Validator provisions are minted on an hourly basis (the first block of a new
+hour).  The annual target of between 7% and 20%. The long-term target ratio of
+bonded tokens to unbonded tokens is 67%.  
```

Comment:
Confusing, there are two concepts here. 
 1) Inflation, which has No target, it is a moving value between 7% and 20% depending on how close the global staked value of atoms is 67%
 2) Target % Bonded Atoms - This is the 67% number

It's confusing to have these sentences one after the other, should distinguish these two percentages a bit more